manuscript/01_introduction.md

```diff
@@ -34,48 +34,45 @@ csl: ./common/templates/nature.csl
 
 # Introduction
 
 ## Tumor Heterogeneity
 
-In 2018, over 1.7 million new cases of cancer were estimated to be diagnosed with over 609,000 of those cases projected to be fatal.[@Smith] One of the primary treatments of cancer consists of chemotherapy, particularly targeted therapies, whereby patients are given drugs that destroy highly-prolific cells to stall cancer growth or eliminate the tumor. Long-term therapeutic efficacy, however, varies significantly due to the vast heterogeneity in intratumor response to therapy [@DiMaio; @DeRoock]. Cell variability can originate from cell-intrinsic factors, such as genomic alterations (i.e. altered nucleotide excision repair and telomere maintenance), and cell-extrinsic factors such as spatial variability in vasculature and environmental stressors causing DNA promoter mutations.[@Feinberg; @Falkenberg; @Inde]. Current drug-screening protocols involve subjecting studied cancer cell lines to variable drug doses and evaluating therapy performance based (1) percent of cells eliminated and (2) dose required for half-maximal effect [@Chauvin; @Zhang_L]. These metrics, however, provide population average tumor response to therapy that fails to consider subpopulation heterogeneity. 
+In 2018, over 1.7 million new cases of cancer were estimated to be diagnosed with over 609,000 of those cases projected to be fatal [@Smith]. One of the primary treatments of cancer consists of chemotherapy, mainly targeted therapies, whereby patients are administered drugs that destroy highly-prolific cells to stall cancer growth or eliminate the tumor. Long-term therapeutic efficacy, however, varies significantly due to the vast heterogeneity in intratumor response to therapy [@DiMaio; @DeRoock]. Cell variability in drug response can originate from cell-intrinsic factors, such as genomic alterations (i.e., altered nucleotide excision repair, telomere maintenance, and copy-number variation), and cell-extrinsic factors such as spatial variability in the surrounding vasculature and environmental stressors causing DNA promoter mutations [@Feinberg; @Falkenberg; @Inde]. Current drug-screening protocols involve subjecting studied cancer cell lines to variable drug doses and evaluating therapy performance by examining (1) the percent of cells eliminated and (2) the dose required for half-maximal effect [@Chauvin; @Zhang_L]. These metrics, however, provide population averaged tumor responses to therapy that fail to consider subpopulation heterogeneity. 
```

Comment:
1.7 million cases is just for the US, right?

Reply:
Yes, just the US; will change. What do you think about the suggestion for the methods Dr. Meyer  @aarmey? Can we make the methods not as intense (mathematically) and leave a lot of it in the supplements/appendix?

Reply:
That sounds reasonable.